src/v3/feedbacks/offset.ts

```diff
@@ -3,19 +3,28 @@ import { OntimeV3 } from '../ontimev3'
 import { feedbackId } from '../../enums'
 import { DangerRed, White } from '../../assets/colours'
 
+enum OffsetState {
+	On = 'on',
+	Behind = 'behind',
+	Ahead = 'ahead',
+	Both = 'both',
+}
+
 export function createOffsetFeedbacks(ontime: OntimeV3): { [id: string]: CompanionFeedbackDefinition } {
 	function offset(feedback: CompanionFeedbackBooleanEvent): boolean {
-		const state = feedback.options.state
+		let state = feedback.options.state
```

Comment:
a small suggestion
you can cast state immediately and then keep state as a constant 
```suggestion
		const state = feedback.options.state as as OffsetState
```

Reply:
good idea - done

Reply:
Good idea - done